ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/EmailIntegrationForm.tsx

```diff
@@ -170,6 +186,17 @@ function EmailIntegrationForm({
         }
     }
 
+    function onUpdateUnauthenticatedChange(isChecked) {
+        if (isChecked) {
+            setStoredUsername(values.notifier.email.username);
+            setFieldValue('notifier.email.username', '');
```

Comment:
@theencee Should we be clearing out the username when switching to unauthenticated, or is a username still required in certain cases?

(More importantly, is it required for the customer who asked for this feature?)

Reply:
Unauth has optional username @vjwilson so safe to clear out